content/blog/infrastructure-as-code-on-hpe-greenlake-using-terraform-–-part-2.md

```diff
@@ -0,0 +1,363 @@
+labels = ["hackshack", "hpedev"]
+
+```
+
+And you can add tags with the following syntax:
+
+```markdown
+# Using tags
+tags = {
+        team  = "HPE Developer"
+        support = "gold"
+  }
+```
+
+Save and apply those changes with **terraform apply**, wait a little and look at the VM details. You can see the labels and the tags in the capture below:
+
+![tags and labels applied to VM](/img/terraform-greenlake-part2-blog-picture3.png "tags and labels applied to VM")
+
+### Use case 3: Get me more disk space
+
+Another typical use case would be to add another disk to a VM, say a data volume for application usage. The syntax for this is the same as you used to create the VM with its *root_vol*, already visible in the Storage details of the VM:
+
+![VM was created with one disk: root_vol](/img/terraform-greenlake-part2-blog-picture4.png "VM was created with one disk: root_vol")
+
+Go ahead and add the following code snippet to your configuration file:
+
+```markdown
+     volume {
+         name         = "data_vol"
+         size         = 25
+         datastore_id = "auto"
+     }
+```
+
+Save the file, apply those changes, wait a little and check the VM storage configuration again:
+
+![VM with two disks](/img/terraform-greenlake-part2-blog-picture5.png "VM with two disks")
+
+### Use case 4: Please snap this VM
+
+Here’s one last use case you can try, which consists of snapshotting the VM. You should use the following Terraform syntax:
+
+```markdown
+   snapshot {
+    name        = "Snap1
+    description = "Snap this VM so we can restart from this state"
+  }
+```
+
+Save the file, apply those changes, wait a little and check the details of the VM in the Backups section once again:
+
+![Snap of VM ready](/img/terraform-greenlake-part2-blog-picture6.png "Snap of VM ready")
+
+## Debugging when things go wrong
+
+In this post, I’ve showed you how to make sure the Terraform configuration file is valid before applying changes using the **terraform validate** command. To see more details during an apply command, you can also enable Terraform debug by simply setting up the TF_DEBUG environment variable:
+
+```markdown
+
+export TF_LOG=DEBUG
+
+```
+
+## What’s next?
+
+In [my first blog post](https://developer.hpe.com/blog/infrastructure-as-code-on-hpe-greenlake-using-terraform/), I covered how to get started with the Terraform provider for HPE GreenLake, I explained how to collect data from the platform and request the creation of a VM instance. In this article, I applied several changes to the Terraform infrastructure configuration file and observe how the desired state is automatically tracked by Terraform and applied to HPE GreenLake.
+
+\-      [ Learn more about Terraform](https://www.terraform.io/)
+
+\-      [ Learn more about HPE GreenLake](https://www.hpe.com/us/en/greenlake.html)
+
+\-      [ Learn more about the HPE GreenLake Terraform provider](https://registry.terraform.io/providers/HewlettPackard/hpegl/0.1.0-beta7)
```

Comment:
Sounds like they moving to HPE ?! Also prefer url without version trailer.
https://registry.terraform.io/providers/HPE/hpegl